packages/sirius-web/frontend/sirius-web-application/src/navigationBar/NavigationBar.tsx

```diff
@@ -66,10 +67,24 @@ const useNavigationBarStyles = makeStyles()((theme) => ({
   },
 }));
 
+export const SiriusWebNavigationBarHome: React.ComponentType<NavigationBarHomeProps> = ({}: NavigationBarHomeProps) => {
```

Comment:
Could you please move the default implementation out of the NavigationBar file (in a dedicated file or where it was before) to keep a clear separation between the consumer of the extension point and one of its implementation.